### PR TITLE
Add docs on monitoring WAL failover

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-10268

Followup to #18511 with the monitoring piece

### CODE REVIEW

**Comment by rmloveland** [comment]
@jbowens this is the content on WAL failover monitoring I'd like to add directly following merge of #18511 

will be rebasing this PR on main once that other PR goes in and then the placement of this content will actually make sense (ideally)

opening in parallel for speed since v24.1 public release is next Monday

**Review by jbowens — APPROVED** [review]
lgtm!

**Comment by rmloveland** [comment]
@florence-crl wanted to add some context b/c this PR is "different", this is a followup to https://github.com/cockroachdb/docs/pull/18511 with just the monitoring info

I split it into a separate PR since that other one is taking a little time to get through review - that one will also be coming your way ASAP for review of course!

File: src/current/v24.1/cockroach-start.md [hunk 1]
```diff
@@ -226,6 +226,21 @@ Field | Description
...
+- `storage.wal.failover.secondary.duration`: Cumulative time spent (in nanoseconds) writing to the secondary WAL directory. Only populated when WAL failover is configured.
+- `storage.wal.failover.primary.duration`: Cumulative time spent (in nanoseconds) writing to the primary WAL directory. Only populated when WAL failover is configured.
+- `storage.wal.failover.switch.count`: Count of the number of times WAL writing has switched from primary to secondary store, and vice versa.
+
+The `storage.wal.failover.secondary.duration` is the primary metric to monitor. You should expect this metric to be `0` unless a WAL failover occurs. If a WAL failover occurs, you probably care about how long it remains non-zero because it provides an indication of the health of the primary store.
+
+You can access these metrics via the following methods:
+
+- The [Custom Chart Debug Page](https://www.cockroachlabs.com/docs/v24.1/ui-custom-chart-debug-page) in [DB Console](https://www.cockroachlabs.com/docs/v24.1/ui-custom-chart-debug-page).
+- By [monitoring CockroachDB with Prometheus](https://www.cockroachlabs.com/docs/v24.1/monitor-cockroachdb-with-prometheus).
```
Comment (florence-crl) [inline src/current/v24.1/cockroach-start.md:242]:
```suggestion
- The [Custom Chart Debug Page]({% link {{ page.version.version }}/ui-custom-chart-debug-page.md %}) in [DB Console]({% link {{ page.version.version }}/ui-overview.md %}).
- By [monitoring CockroachDB with Prometheus]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}).
```

**Review by florence-crl — APPROVED** [review]
1 suggestion to format links in liquid. Otherwise LGTM!

File: src/current/v24.1/cockroach-start.md (hunk 1 above)
Comment (rmloveland) [reply src/current/v24.1/cockroach-start.md:242]: ah thanks! completely forgot that one

**Pull request merged**